From 48a7852db428cdac57948a0d6b5b040b7a081782 Mon Sep 17 00:00:00 2001
From: Dong Du <ddnirvana1@gmail.com>
Date: Mon, 24 May 2021 15:14:54 +0800
Subject: [PATCH] Enable build id for elf files

Signed-off-by: Dong Du <ddnirvana1@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d6f097d..347318b 100644
--- a/Makefile
+++ b/Makefile
@@ -228,7 +228,7 @@ ASFLAGS		+=	$(firmware-asflags-y)
 
 ARFLAGS		=	rcs
 
-ELFFLAGS	+=	-Wl,--build-id=none -N -static-libgcc -lgcc
+ELFFLAGS	+=	-N -static-libgcc -lgcc
 ELFFLAGS	+=	$(platform-ldflags-y)
 ELFFLAGS	+=	$(firmware-ldflags-y)
 
-- 
2.31.1

