From 458c01801bb9f0084d47ef421809693517c48ccf Mon Sep 17 00:00:00 2001
From: zhangxuzhou <zhangxuzhou4@huawei.com>
Date: Mon, 3 Aug 2020 16:58:44 +0800
Subject: [PATCH] Enable build id for elf files

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ccf8ea3..025ccf8 100644
--- a/Makefile
+++ b/Makefile
@@ -191,7 +191,7 @@ ASFLAGS		+=	$(firmware-asflags-y)
 
 ARFLAGS		=	rcs
 
-ELFFLAGS	+=	-Wl,--build-id=none -N -static-libgcc -lgcc
+ELFFLAGS	+=	-N -static-libgcc -lgcc
 ELFFLAGS	+=	$(platform-ldflags-y)
 ELFFLAGS	+=	$(firmware-ldflags-y)
 
-- 
2.26.2

